package/src/utils/utils.ts

```diff
@@ -24,12 +24,11 @@ import type {
   SuggestionUser,
 } from '../contexts/suggestionsContext/SuggestionsContext';
 import { compiledEmojis, Emoji } from '../emoji-data';
-import type { IconProps } from '../icons/utils/base';
 import type { TableRowJoinedUser } from '../store/types';
 import type { DefaultStreamChatGenerics, ValueOf } from '../types/types';
 
 export type ReactionData = {
-  Icon: React.ComponentType<IconProps>;
+  Icon: React.ComponentType;
   type: string;
```

Comment:
is this change necessary?, seems like its making the type looser now..

Reply:
I saw how this is used https://github.com/GetStream/stream-chat-react-native/blob/54cccbad154f9a4acf057c7892bc8bc52c0fe58e/package/src/components/Message/MessageSimple/ReactionList.tsx#L37-L38

Reply:
intuiverly it seems like this removal should not be done to me

Reply:
I think this is better

```
Icon: React.ComponentType<Partial<IconProps>>;
```

Supports just a normal Text component and also integrators get to add support for the icon props if they want for SVG override